GVFS/GVFS/RepairJobs/RepairJob.cs

```diff
@@ -37,7 +37,7 @@ public enum FixResult
 
         protected ITracer Tracer { get; }
         protected TextWriter Output { get; }
-        protected GVFSEnlistment Enlistment { get; }
+        protected GVFSEnlistment Enlistment { get; set; }

```

Comment:
Yea I definitely think we should keep the property readonly. In general, readonly state is much easier to reason about than mutable state.